.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   main-go:
-    name: CI [Go]
+    name: [Go] CI 
```

Comment:
don't change these names as then we need to adapt the build validation branch rules again

Reply:
Ok. I will change it back